asmrun/arm.S

```diff
@@ -17,7 +17,7 @@
 
         .syntax unified
         .text
-#if defined(SYS_linux_eabihf) && defined(MODEL_armv6)
+#if (defined(SYS_linux_eabihf) || defined(SYS_netbsd)) && defined(MODEL_armv6)
```

Comment:
You would still have to define `SYS_linux`, `SYS_netbsd`, `SYS_solaris`, etc. for platform-specific stuff. I'm arguing for replacing `SYS_linux_eabi` and `SYS_linux_eabihf` with `SYS_linux`, `ABI_eabi`, and `ABI_eabihf` (or something equivalent.)